src/components/stepforms/schemas/rollup.ts

```diff
@@ -0,0 +1,95 @@
+import { addNotInColumnNamesConstraint, StepFormType } from './utils';
+
+const schema = {
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  title: 'Rollup step',
+  type: 'object',
+  properties: {
+    name: {
+      type: 'string',
+      enum: ['rollup'],
```

Comment:
I think there is a `const` property in json schema that fits this case better :) 
https://json-schema.org/understanding-json-schema/reference/generic.html#constant-values
It's from draft 6 and we're using draft 7 here, so it shouldn't be a problem.